### PR TITLE
Fixes problems with fragment generation on volume zarrs

### PR DESCRIPTION
**Pull request by caethan**

### CODE REVIEW

File: volume_zarr.py [hunk 1]
```diff
@@ -0,0 +1,636 @@
...
+            return self.data[kslice, jslice, it].T
+        raise ValueError
+
+    def ijIndexesInPlaneOfSlice(self, axis):
+        return ((1,2), (0,2), (0,1))[axis]
+
+    def ijInPlaneOfSlice(self, axis, ijkt):
+        #inds = self.ijIndexesInPlaneOfSlice(axis)
+        #return(ijkt[inds[0]], ijkt[inds[1]])
+        return (self.max_width, self.max_width)
```
Comment (KhartesViewer) [inline volume_zarr.py]:
```suggestion
        inds = self.ijIndexesInPlaneOfSlice(axis)
        x,y = ijkt[inds[0]], ijkt[inds[1]]
        return (min(x, self.max_width), min(y, self.max_width))
```

Comment (KhartesViewer) [reply volume_zarr.py]: Not sure if I am doing this correctly?  I'm new at using github for code review.  Let me know if you prefer a different approach.  Anyway, the change to ijInPlaneOfSlice is to fix an alignment problem at the left and top sides of the volume.

**Comment by KhartesViewer** [comment]
Hi!  I've finally started going through your PR in detail.
I really like being able to move through the entire scroll! 
I downloaded your branch, managed to add the necessary libraries to conda (I'm working in the Windows world), and got it to run.
I'd like to merge your code into my repository, at a time when you feel it is in a stable state, so let me know!
In the meantime, if you don't mind, I'll send you bug reports (perhaps with fixes) as I encounter them, if that is ok with you.

**Comment by caethan** [comment]
Sure thing!  The main branch should be stable at this point.  I’m working
on some extensive modifications in another branch that are still in
progress.

**Comment by KhartesViewer** [comment]
Good!  But the heading of this PR reads:
[caethan](https://github.com/caethan) wants to merge 16 commits into [KhartesViewer:main](https://github.com/KhartesViewer/khartes/tree/main) from [caethan:bno/volume_annotation](https://github.com/caethan/khartes/tree/bno/volume_annotation)

Should the PR be pulling from your main branch instead?

**Comment by caethan** [comment]
Bother, yes, I probably need to fix that.  Give me a bit to put the kids in
bed and I can get on that.

Give the annotation branch a try though, it’s got some interesting work in
it.

**Comment by KhartesViewer** [comment]
No problem, no rush!  I've been starting to look at the experimental branch; the first step was to figure out how to import pymaxflow.  So far I have only looked at the volume_zarr.py stuff, which is pretty interesting for me.  I was impressed at how you were able to integrate it by making it behave like the Volume class.
Anyway, I'm guessing that the cleanest way to deal with the PR is to close this one and open a new one on the main branch, but I'll leave that to you to decide!

**Pull request closed without merging**